.github/workflows/ci_compliance.yml

```diff
@@ -15,12 +14,48 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: actions/checkout@v3
+
+      - uses: actions-rs/toolchain@v1.0.7
+        id: toolchain
+        with:
+          toolchain: stable
+          override: true
+
+      - uses: camshaft/rust-cache@v1
```

Comment:
There is also a `cache` operation in the duvet action and I am not sure if the caching will mess with consecutive runs. From my own testing it didnt seem to interact. @camshaft

Reply:
Why do we need to duplicate all of these steps from the workflow? Why not just run the check after it runs?

Reply:
I wanted to avoid the upload to s3 (part of the duvet action) if the checks fails.